shells/pipes-shell/source/pipe.js

```diff
@@ -50,9 +52,74 @@ const populateDispatcher = (dispatcher, composerFactory, storage, context) => {
     pec: async (msg, tid, bus) => {
       return await pec(msg, tid, bus);
     },
+    ingest: async (msg, tid, bus) => {
```

Comment:
i didn't realize we're bringing back `ingest` message and all the related changes, including sharing.

Reply:
For various demos and examples to work we need a solution here. The new stuff is not ready yet, as I understand it, and this is harmless since it's easily removed when it's truly not useful anymore.